napari/_pydantic_compat.py

```diff
@@ -1,65 +1,39 @@
-try:
-    # pydantic v2
-    from pydantic.v1 import (
-        BaseModel,
-        BaseSettings,
-        Extra,
-        Field,
-        PositiveInt,
-        PrivateAttr,
-        ValidationError,
-        color,
-        conlist,
-        constr,
-        errors,
-        main,
-        parse_obj_as,
-        root_validator,
-        types,
-        utils,
-        validator,
-    )
-    from pydantic.v1.env_settings import (
-        EnvSettingsSource,
-        SettingsError,
-        SettingsSourceCallable,
-    )
-    from pydantic.v1.error_wrappers import ErrorWrapper, display_errors
-    from pydantic.v1.fields import SHAPE_LIST, ModelField
-    from pydantic.v1.generics import GenericModel
-    from pydantic.v1.main import ClassAttribute, ModelMetaclass
-    from pydantic.v1.utils import ROOT_KEY, sequence_like
-except ImportError:
-    # pydantic v1
-    from pydantic import (
-        BaseModel,
-        BaseSettings,
-        Extra,
-        Field,
-        PositiveInt,
-        PrivateAttr,
-        ValidationError,
-        color,
-        conlist,
-        constr,
-        errors,
-        main,
-        parse_obj_as,
-        root_validator,
-        types,
-        utils,
-        validator,
-    )
-    from pydantic.env_settings import (
-        EnvSettingsSource,
-        SettingsError,
-        SettingsSourceCallable,
-    )
-    from pydantic.error_wrappers import ErrorWrapper, display_errors
-    from pydantic.fields import SHAPE_LIST, ModelField
-    from pydantic.generics import GenericModel
-    from pydantic.main import ClassAttribute, ModelMetaclass
-    from pydantic.utils import ROOT_KEY, sequence_like
+"""
+This module was added for compatibility with pydantic v1 and v2.
+I think that we should keep them until we stop using pydantic.v1 module, and fully use
+pydantic 2 module.
```

Comment:
```suggestion
This module provides compatibility between pydantic v1 and v2.

Keep using this compatibility module until we stop using any pydantic v1 API functionality.
This can be removed when everything has been migrated to pydantic v2.
```

Reworded to provide context for future devs.